packages/recoil/recoil_values/Recoil_atom.js

```diff
@@ -292,6 +293,23 @@ function baseAtom<T>(options: BaseAtomOptions<T>): RecoilState<T> {
         return getLoadable(recoilValue).toPromise();
       }
 
+      function observe<S>(
```

Comment:
Naming is always hard..  I suppose might have made sense for this to be `onSet()`, and that could have been `onSetSelf()` or something; but too late to make that change now.  Hmm, maybe this could be `onSetValue()`...

Reply:
I'm open to changing this, that suggestion seems to at least rhyme with the current naming scheme. I'll push an update with your suggestion.